project-1/src/App.js

```diff
@@ -7,9 +7,7 @@ function App() {
   return (
     <div className="app-container">
       <h1 className="app-heading">Welcome to My Home Page!</h1>
-      <p className="app-message">
-        Thank you for visiting. We're glad you're here!
-      </p>
+      <p className="app-message">We thank you for visiting. Come back soon!</p>
```

Comment:
Maybe this should be a more meaningful welcome message

Reply:
This issue has been resolved the new message is better than the initial text